backend/search/models.py

```diff
@@ -76,7 +70,7 @@ def check_results(self) -> bool:
 
     def get_results(self) -> ResultSet:
         """Return results as a dict"""
-        cache_filename = str(self._get_cache_path(False))
+        cache_filename = str(self._get_cache_path())
```

Comment:
Good of course, but those lines can be shortened to `results = self._get_cache_path().read_text()`.

Reply:
cool, I didn't know about `.read_text()`